### PR TITLE
php 7.4.8

### PR DESCRIPTION
**Pull request by derrabus**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/php.rb [hunk 1]
```diff
@@ -2,9 +2,8 @@ class Php < Formula
   desc "General-purpose scripting language"
   homepage "https://www.php.net/"
   # Should only be updated if the new version is announced on the homepage, https://www.php.net/
-  url "https://www.php.net/distributions/php-7.4.7.tar.xz"
-  mirror "https://fossies.org/linux/www/php-7.4.7.tar.xz"
```
Comment (dawidd6) [inline Formula/php.rb:6]: https://fossies.org/linux/www/php-7.4.8.tar.xz

Comment (SMillerDev) [reply Formula/php.rb:6]: Or rather, please don't remove the mirror.

Comment (derrabus) [reply Formula/php.rb:6]: Done. For the future: Shall I keep the mirror even if the mirror hasn't been updated yet? It did not have the file when I opened the PR.

**Review by chenrui333 — APPROVED** [review]

File: Formula/php.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/php.rb:6]: yeah, keep the mirror, it might be slow ftp server update.

**Pull request closed without merging**